terraform/aptos-node/aws/variables.tf

```diff
@@ -142,7 +142,7 @@ variable "helm_enable_validator" {
 
 variable "utility_instance_type" {
   description = "Instance type used for utilities"
-  default     = "t3.medium"
+  default     = "t4g.medium"
```

Comment:
Are we going to run prometheus on this? I saw it sometimes consume a lot of RAM and get OOM killed.

Reply:
single node deployment doesn't have this problem, you can override the instance type for the testnet terraform